src/test/regress/spec/isolation_multiuser_locking.spec

```diff
@@ -1,3 +1,7 @@
+// Two alternative test outputs:
```

Comment:
Can you also add the relevant PG commit in the comments, and also explain why the output is different between PG16 and PG17 (i.e. explanation about maintain privileges)